byzcoin/statetrie_test.go

```diff
@@ -2,7 +2,18 @@ package byzcoin
 
 import (
 	"bytes"
+	"encoding/hex"
+	"math/big"
+	"strings"
 	"testing"
+	"time"
+
+	"go.dedis.ch/onet/v3"
+
+	"go.dedis.ch/kyber/v3/util/random"
+
+	"github.com/ethereum/go-ethereum/common/math"
```

Comment:
Nope

Reply:
Thought about it - as bevm already uses it, I thought it's OK to use it here, too...

Reply:
didn't see the `(*big.Int).Exp`